machines/xcelium_dump.tcl

```diff
@@ -0,0 +1,4 @@
+database -open dump -shm
```

Comment:
Xcelium doesn't seem to have a command line interface, so we're forced to use TCL here